fdbserver/TLogServer.actor.cpp

```diff
@@ -46,6 +46,7 @@
 #include "flow/Histogram.h"
 #include "flow/DebugTrace.h"
 #include "flow/actorcompiler.h" // This must be the last #include.
+#include "flow/network.h"
```

Comment:
This is after the actor compiler include which I think causes problems in some scenario, hence the comment on the line above.